Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/V3Layer.h

```diff
@@ -0,0 +1,455 @@
+    static const Double_t sIBConnectAFitZLen;  ///< IB ConnectorA Fitting Z len
+    static const Double_t sIBConnectAFitZOut;  ///< IB ConnectorA Fitting Z Out
+    static const Double_t sIBConnPlugInnerD;   ///< IB Connector Plug int diam
+    static const Double_t sIBConnPlugTotLen;   ///< IB Connector Plug tot le
+    static const Double_t sIBConnPlugThick;    ///< IB Connector Plug thickness
+
+    static const Double_t sIBStaveHeight;      ///< IB Stave Total Y Height
+
+    // Outer Barrel Parameters
+    static const Int_t sOBChipsPerRow; ///< OB chips per row in module
+    static const Int_t sOBNChipRows;   ///< OB chip rows in module
+
+    static const Double_t sOBHalfStaveWidth;    ///< OB Half Stave Width
+    static const Double_t sOBModuleWidth;       ///< OB Module Width
+    static const Double_t sOBModuleGap;         ///< Gap between OB modules
+    static const Double_t sOBChipXGap;          ///< Gap between OB chips on X
+    static const Double_t sOBChipZGap;          ///< Gap between OB chips on Z
+    static const Double_t sOBFlexCableAlThick;  ///< Thickness of FPC Aluminum
+    static const Double_t sOBFlexCableKapThick; ///< Thickness of FPC Kapton
+    static const Double_t sOBBusCableAlThick;   ///< Thickness of Bus Aluminum
+    static const Double_t sOBBusCableKapThick;  ///< Thickness of Bus Kapton
+    static const Double_t sOBCarbonPlateThick;  ///< OB Carbon Plate Thickness
+    static const Double_t sOBColdPlateThick;    ///< OB Cold Plate Thickness
+    static const Double_t sOBGlueThick;         ///< OB Glue total Thickness
+    static const Double_t sOBModuleZLength;     ///< OB Chip Length along Z
+    static const Double_t sOBHalfStaveYTrans;   ///< OB half staves Y transl.
+    static const Double_t sOBHalfStaveXOverlap; ///< OB half staves X overlap
+    static const Double_t sOBGraphiteFoilThick; ///< OB graphite foil thickness
+    static const Double_t sOBCoolTubeInnerD;    ///< OB cooling inner diameter
+    static const Double_t sOBCoolTubeThick;     ///< OB cooling tube thickness
+    static const Double_t sOBCoolTubeXDist;     ///< OB cooling tube separation
+
+    static const Double_t sOBSpaceFrameWidth;   ///< OB Space Frame Width
+    static const Double_t sOBSpaceFrameTotHigh; ///< OB Total Y Height
+    static const Double_t sOBSFrameBeamRadius;  ///< OB Space Frame Beam Radius
+    static const Double_t sOBSpaceFrameLa;      ///< Parameters defining...
+    static const Double_t sOBSpaceFrameHa;      ///< ...the V side shape...
+    static const Double_t sOBSpaceFrameLb;      ///< ...of the carbon...
+    static const Double_t sOBSpaceFrameHb;      ///< ...OB Space Frame
+    static const Double_t sOBSpaceFrameL;       ///< OB SF
+    static const Double_t sOBSFBotBeamAngle;    ///< OB SF bottom beam angle
+    static const Double_t sOBSFrameBeamSidePhi; ///< OB SF side beam angle
+
+  ClassDef(V3Layer, 0) // ITS v3 geometry
```

Comment:
please use ClassDefOverride here (there should be a compiler warning about that)